src/llamafactory/model/loader.py

```diff
@@ -82,6 +82,9 @@ def load_tokenizer(model_args: "ModelArguments") -> "TokenizerModule":
             padding_side="right",
             **init_kwargs,
         )
+    except Exception as e:
+        logger.error("Failed to load tokenizer. Error: {}".format(e))
+        raise e
```

Comment:
`raise OSError("Failed to load tokenizer") from e`

Reply:
addressed. but i do not think it is OSError :)